expected-result-generator/bin/generate_results.py

```diff
@@ -10,8 +10,11 @@
 import argparse
 import subprocess
 import os
+import glob
 
-CLASSPATH = ".:../build/libs/expected-result-generator-all.jar"
+jar_file_list = glob.glob("../build/libs/expected-result-generator-all*.jar")
```

Comment:
this probably does not belong to this change... Please mention about that in commit message